taglib/riff/riffutils.h

```diff
@@ -44,13 +44,7 @@ namespace TagLib
         if(name.size() != 4)
           return false;
 
-        for(auto it = name.begin(); it != name.end(); ++it) {
-          const int c = static_cast<unsigned char>(*it);
-          if(c < 32 || 127 < c)
-            return false;
-        }
-
-        return true;
+        return std::none_of(name.begin(), name.end(), [](char c) { return c < 32 || 127 < c; });
```

Comment:
Is `char` the correct type for `c`?

Reply:
Original code casts it to unsigned. Maybe it'd be good to do that here too.